src/Utility/TypeExtensions.cs

```diff
@@ -89,24 +90,34 @@ internal static object ToObject(this TypedData data)
                 case TypedData.DataOneofCase.None:
                     return null;
                 default:
-                    return new InvalidOperationException("Data Case was not set.");
+                    return new InvalidOperationException("DataCase was not set.");
             }
         }
 
-        private static JsonSerializerSettings setting =
-            new JsonSerializerSettings { TypeNameHandling = TypeNameHandling.None, MaxDepth = 3 };
-        private static object DeserializeJson(string json)
+        // PowerShell NuGet packages only have 'System.Management.Automation.dll' as the ref assembly, and thus types from other powershell assemblies
+        // cannot be used directly in an application that reference the PowerShell NuGet packages.
+        // Here we need to use 'Microsoft.PowerShell.Commands.JsonObject' from 'Microsoft.PowerShell.Commands.Utility'. Due the above issue, we have to
+        // use reflection to call 'JsonObject.ConvertFromJson(...)'.
+        private const string UtilityAssemblyFullName = "Microsoft.PowerShell.Commands.Utility, Version=6.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35";
```

Comment:
can you put this const inside the function?

Reply:
Sure, will do.